src/scripts/OSFramework/GlobalEnum.ts

```diff
@@ -352,4 +352,15 @@ namespace OSFramework.GlobalEnum {
 	export enum NullValues {
 		Time = '00:00:00',
```

Comment:
@rugoncalves Just a doubt. Shouldn't this be a constant instead of an enumerator?

Reply:
I haven't changed this. However, I think it was made to then have other null values, such as nulldate and others. Good question why that isn't there yet?!..

Reply:
@joselrio any insight on this?